src/app.js

```diff
@@ -21,13 +22,16 @@ export function app(app) {
     })
   }
 
-  if (document.readyState[0] !== "l") {
-    load()
-  } else {
-    addEventListener("DOMContentLoaded", load)
-  }
+  return emit("loaded", emit, render(state, view))
```

Comment:
isn't defer supported by like 95%+ of browsers now?  http://caniuse.com/#search=defer

stupid Opera Mini dragging their feet on so much.